docs/architecture/data-plane-osd-cluster-dynamic-scaling.md

```diff
@@ -6,18 +6,181 @@ This documents explains how Data Plane OSD Cluster Dynamic Scaling functionality
 has been implemented.
 
 Data Plane OSD Cluster Dynamic Scaling functionality currently deals with:
-* Autoscaling of compute nodes of an OSD cluster
-* Prewarming of compute nodes 
+* Autoscaling of nodes of an OSD cluster
+* Prewarming of nodes 
 * OSD Cluster creation and deletion
 
-## Autoscaling of compute nodes of an OSD cluster
+## Autoscaling of worker nodes of an OSD cluster
 
-<!---TODO -->
+Autoscaling of worker nodes of an OSD cluster is done by leveraging the [Cluster Autoscaler](https:docs.openshift.com/container-platform/4.9/machine_management/applying-autoscaling.html) as described in https:architecture.appservices.tech/ap/15/#autoscaling-of-nodes. For Manual clusters, this has to be enabled manually. Worker node autoscaling is enabled by default for all clusters that are created dynamically by the Fleet manager
 
-## Prewarming of compute nodes 
+## Prewarming of worker nodes 
 
-<!---TODO -->
+Data Plane Worker Node Prewarming helps speed up the allocation of Kafka instances in the Data Planes managed by the control plane in some scenarios. It
+is achieved by reserving resources in the Data Planes. The reservation of resources is specified in the form of reserved Kafka instances of a given
+size and it is done on a per Kafka instance type basis.
+
+When enabling and configuring Data Plane Node Prewarming for a given Kafka instance type, each Data Plane managed by the Control Plane maintains the
```

Comment:
Is it worker node prewarming or just node prewarming? :thinking:

Reply:
being strict, worker

Reply:
+1 It is worker node prewarming.

I’ll double check all occurrences and update it.

Reply:
I've updated this part